### ISSUE TITLE
Not an issue but I understand the "malicious" part behind the code

### ISSUE THREAD

**Issue by chaikwoozy** [HarmOnGitHub/rylix-is-a-skid#3]
So he can download an discord webhook to your device by hiding it in the code within the hacks. This webhook can get your PC/laptop/phone information, information inside your device, and your personal info.

I have seen on GitHub discord webhooks with install code you can put inside an website or an part of code.

**Comment by HarmOnGitHub** [comment]
No, he uses the webhook to send your information off to himself.

**Comment by chaikwoozy** [comment]
Oh okayyyyy.
